app/javascript/components/key-performance-indicators/components/stacked-percentage-bar/stacked-percentage-bar-meter.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
I would maybe just move this component to key-perforance-indicators's components dir in a folder called stacked-percentage-bar-meter and call this file component.jsx same goes for the corresponding label component